### ISSUE TITLE
Integrate Cloudflare API for domain management?

### ISSUE THREAD

**Issue by cammclain** [OTRLabs/CryptCloud-Platform#8]
Use the cloudflare api to manage domains from within CryptCloud

This has 2 different functions 

CryptCloud as a whole

TL;DR: I am sure being able to manage multiple domains is a good idea. I imagine it as something similar to Vercel but with an “offensive” twist

Individual CryptHub Installations

It is beneficial for security reasons to have a wide variety of domains available 

This is for a few reasons:

1 General Cybersecurity 

Some platforms we are integrating with do not like it when you use subdomains of one central domain for all your services

For example matrix wants you to host your home server on a unique domain

2 Offensive Advantages

Being able to “shape shift” & rotate domains would be highly advantageous during offensive operations 

For example being able to disable a domain after it has been used for phishing or as a C2 listener would be more than ideal, meaning it would be harder to investigate for the blue team
